<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>688</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>437</width>
    <height>688</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>437</width>
    <height>688</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Kayıt Ol</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-10</y>
     <width>451</width>
     <height>691</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(4, 18, 27);</string>
   </property>
   <widget class="QPushButton" name="push_button_kayit_ol">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>460</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>17</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(231, 231, 231);
border: 2px solid rgb(65, 199, 199);
padding: 5px;
border-radius: 10px;
opacity: 200;\n</string>
    </property>
    <property name="text">
     <string>Kayıt Ol</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="line_edit_username">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(231, 231, 231);

border: None;
border-bottom-color: white;
border-radius: 10px;
padding: 0 8px;
background: rgb(4, 18, 27);
selection-background-color: darkgray;</string>
       </property>
       <property name="placeholderText">
        <string>Kullanıcı Adı</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="styleSheet">
        <string notr="true">border: 2px solid white;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::HLine</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>380</y>
      <width>301</width>
      <height>44</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLineEdit" name="line_edit_password">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(231, 231, 231);

border: None;
border-bottom-color: white;
border-radius: 10px;
padding: 0 8px;
background: rgb(4, 18, 27);
selection-background-color: darkgray;</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>Şifre</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="styleSheet">
        <string notr="true">border: 2px solid  rgb(65, 199, 199);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::HLine</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>301</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLineEdit" name="line_edit_mail">
       <property name="font">
        <font>
         <family>Verdana</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(231, 231, 231);

border: None;
border-bottom-color: white;
border-radius: 10px;
padding: 0 8px;
background: rgb(4, 18, 27);
selection-background-color: darkgray;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Mail</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="styleSheet">
        <string notr="true">border: 2px solid white;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::HLine</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="push_button_cikis">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>560</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>17</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(231, 231, 231);
border: 2px solid rgb(65, 199, 199);
padding: 5px;
border-radius: 10px;
opacity: 200;\n</string>
    </property>
    <property name="text">
     <string>Çıkış</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>231</width>
      <height>231</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/icons/deepfake.png);
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>390</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/icons/lock_32x32.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/icons/mail_32x32.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/newPrefix/icons/user_32x32.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="push_button_geri">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>510</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
      <pointsize>17</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(231, 231, 231);
border: 2px solid rgb(65, 199, 199);
padding: 5px;
border-radius: 10px;
opacity: 200;\n</string>
    </property>
    <property name="text">
     <string>Geri</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../../OneDrive/Masaüstü/Yeni klasör (5)/modern-login-main/icons.qrc"/>
 </resources>
 <connections/>
</ui>
